modules/passkey/.env.example

```diff
@@ -5,5 +5,3 @@ PK=""
 CUSTOM_NODE_URL=""
 # Used for etherscan verification
 ETHERSCAN_API_KEY=""
-# Used for deploying contracts
-DEPLOYMENT_ENTRY_POINT_ADDRESS="0x0000000071727De22E5E9d8BAf0edAc6f37da032"
```

Comment:
No longer needed since we don't do the `SafeSignerLaunchpad` deployment anymore.